orchestra/static/orchestra/common/components/checklist/js/directives.js

```diff
@@ -21,7 +21,7 @@
           'comments': [{
             'author': 'jane',
             'timestamp': new Date(new Date() - 3 * 60 * 60 * 1000),
-            'text': "The hero image you've chosen doesn't fit " +
+            'text': 'The hero image you\'ve chosen doesn\'t fit ' +
```

Comment:
Sure, I added `"validateQuoteMarks": { "mark": "'", "escape": true },` to the rules and reverted the relevant changes!